build.cake

```diff
@@ -735,8 +775,9 @@ Task("TestAll")
     .IsDependentOn("TestDriver")
 	.IsDependentOn("TestAddins")
 	.IsDependentOn("TestV2Driver")
-	.IsDependentOn("TestConsole");
-
+	.IsDependentOn("TestConsole")
+    .IsDependentOn("TestCompactFramework");
```

Comment:
Putting this in the TestAll target and not the Test target means that when we specify -t Test -Framework net-cf-3.5, the tests won't be run. However, the point is moot if you drop the TestCompactFramework task.